Screens/LineupForm.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react'
+import {
+  Button,
+  Dimensions,
+  Picker,
+  StyleSheet,
+  Text,
+  View,
+  Image,
+} from 'react-native'
+
+const { width, height } = Dimensions.get('window')
+
+class LineupForm extends Component {
+  static navigationOptions = ({ navigation }) => ({
+    title: `${navigation.state.params.title}`,
+  })
+
+  constructor(props) {
+    super(props)
+
+    propState = props.navigation.state.params.props
+    this.state = {
+      codex:         propState.codex,
+      gender:        propState.gender,
+      temperature:   propState.temperature,
+      precipitation: propState.precipitation,
+      course:        propState.course,
+      racers:        propState.racers,
+      position:      propState.position,
+      images: [],
+      lane1: '',
+      lane2: '',
+      lane3: '',
+      lane4: '',
+      lane5: '',
+      lane6: '',
+    }
+  }
+
+  setImages = (data) => {
+    this.setState({
+      images:
+        [...this.state.images,
+          {
+            source: data.mediaUri
+          }
+        ]
+      })
+    console.log(this.state.images)
+  }
+
+  renderImages() {
+    if (this.state.images.length < 1) {
+      return (
+        <Text style={styles.imageText}>
+          Use the camera to capture images
+        </Text>
+      )
+    } 
+    else {
+      return (
+        <View style={styles.imageGallery}>
+          <Image
+            style={styles.image}
+            source={{uri: this.state.images[0].source}}
```

Comment:
>[0] [](start = 43, length = 3)

Not sure what the expected behavior is here, but at first glance it look like your always grabbing the first photo that was taken, but then above you're appending the most recent photo.